### ISSUE TITLE
[Feature Request] Timeout Backup Success notifications

### ISSUE THREAD

**Issue by RobertZenz** [NeoApplications/Neo-Backup#655]
**Is your feature request related to a problem? Please describe it.**

I have a daily schedule to run backups during the night, this always leaves the "Backup OK" notification for me to dismiss in the morning. I only really care if the backup failed, though.

**Describe the primary solution**

It would be nice if that notification could timeout on its own.

**Comment by thatso** [comment]
Another option would be to prevent showing success notifications of scheduled backups all together. Currently, I have completely disabled Neo Backup notifications to not get bothered every morning. Of course this has the downside that I also will not get any warnings of failed backups.

**Comment by hg42** [comment]
what kind of "bothered"?
swiping it away? (well...)
or sounds? (just make it silent)

what is a backup worth to you?

if the positive feedback is suppressed, you think, everything is ok,  if there is **no** notification.
But maybe NB didn't do anything, the schedule did not run at all or a crash finishes NB each night before it can send a negative notification.

A possible strategy could be, to replace a notification of a schedule with a new notification of the same schedule.
Though, I think, you would easily take the older one for the current one (by not looking at the age of the notification), especially if you are not interested in your backup^H^H^H^H^H^H^H^H^H^H^H^H the positive feedback.

Timeout: would be possible, but what time?
but NB would have to set an alarm only to cancel the notification.

**Comment by thatso** [comment]
It's really not about the worth of the backup but about avoiding annoyances. Your arguments are perfectly valid, yet some people prefer to only be informed if something went wrong.
How about independently checking if the last backup ran successfully or at all and only then showing a negative notification?

**Comment by hg42** [comment]
well, if you don't control success of your backup, it's not worth a penny...
In fact a professional backup strategy would even restore the backup from time to time to see if it really, really works.

what you want, is like telling your girlfriend (or whatever) that starts a journey
"please only contact me, when something's wrong"
(so, when she's on a lonesome beach with broken legs and the phone battery is exhausted or more drastically when she's dead)

If you want to avoid the "work" of swiping away, you may need some automation tool, but you should carefully think about strategy.
E.g. something like this suggestion would be a good solution:
https://github.com/NeoApplications/Neo-Backup/issues/307
Even then the service could fail to message you. Or it's going to your spam folder or similar.

Basically all **push** solutions can fail the same way.
The bigger the distance, the bigger is the chance to fail notifying you...

From this POV even NBs notification is subject to fail. The best would be to write a log and the user has to read the log if everything worked...
LOL, just kidding, but in principle it's like this.

**Comment by thatso** [comment]
You seem to take this way too personal. In fact, your idea with the log is not that far-fetched. Android already has logging facilities. What about creating entries like "backup started", "backup successful", "backup failed" etc. which can be checked with other tools? Or even the possibility to log these events to an external syslog server?
